coldfront/core/user/forms.py

```diff
@@ -95,25 +95,35 @@ def clean_email(self):
             raise forms.ValidationError(mark_safe(message))
         return email
 
-    def clean_middle_name(self):
-        cleaned_data = super().clean()
-        self.middle_name = cleaned_data.pop('middle_name', '')
-        return cleaned_data
-
     def clean_phone_number(self):
         cleaned_data = super().clean()
         self.phone_number = cleaned_data.pop('phone_number', '')
         return cleaned_data
 
+    def clean_first_name(self):
+        cleaned_data = super().clean()
+        name = self.cleaned_data.pop('first_name', '')
+        return name.title()
```

Comment:
Cool, did not know this method existed! It does exactly what we want in many different cases.